### DISCUSSION TITLE
Remaining indirect deprecation notices

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#44307] [Q&A] [answered]
When I run `php ./vendor/bin/phpunit` command, test is ok but display `Remaining indirect deprecation notices`

```
Testing started at 12:18 PM ...
PHPUnit 9.5.10 by Sebastian Bergmann and contributors.

Testing C:\projects\karaka\tests\Controller


Time: 00:00.619, Memory: 28.00 MB

OK (1 test, 1 assertion)

Remaining indirect deprecation notices (1)

  1x: The "Symfony\Bridge\Doctrine\Logger\DbalLogger" class implements "Doctrine\DBAL\Logging\SQLLogger" that is deprecated Use {@link \Doctrine\DBAL\Logging\Middleware} or implement {@link \Doctrine\DBAL\Driver\Middleware} instead.
    1x in ProductControllerTest::testProduct from App\Tests\Controller

Process finished with exit code 0
```

ProductControllerTest.php

```
<?php

namespace App\Tests\Controller;

use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;

class ProductControllerTest extends WebTestCase
{
    public function testProduct(): void
    {
        $client = static::createClient();
        $crawler = $client->request('POST', '/product');

        $this->assertResponseIsSuccessful();
    }
}
```

How can I fix this warning?

**Comment by mbabker** [comment, marked as answer]
You don't.  It'll get fixed in a combination of the `symfony/doctrine-bridge` and `doctrine/doctrine-bundle` packages.

**Reply by karaka200** [reply]
> You don't. It'll get fixed in a combination of the `symfony/doctrine-bridge` and `doctrine/doctrine-bundle` packages.

So I do not pay attention to this warning now?

**Reply by mbabker** [reply]
Yeah.  Essentially, if the deprecation is fully coming from vendor code, it's reasonably safe to ignore unless you're trying to upgrade one of the vendors involved to a new major version (which in this case would be DBAL 4.0, which isn't a thing yet).
